### PR TITLE
OFMCC-6476-Create Funding Envelope Change Request through Funding Tile

### PR DESCRIPTION
**Pull request by Panika-cgi**
Implements: 
[https://eccbc.atlassian.net/browse/OFMCC-6476](https://eccbc.atlassian.net/browse/OFMCC-6476)

- Added test for Create Funding envelope change request through the funding tile.

### CODE REVIEW

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingAllocationTab.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public PortalFundingAllocationTab(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+        wait = new WebDriverWait(driver, Duration.ofMillis(10000));
+    }
+
+
+    public void ClickReallocateFunds() throws InterruptedException {
+        try {
+            wait.until(ExpectedConditions.visibilityOf(requestToReallocateFunds));
```
Comment (Rylan-cgi) [inline selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingAllocationTab.java:30]: We do not need this

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingEnvelopeChangeRequest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+        wait = new WebDriverWait(driver, Duration.ofMillis(10000));
+    }
+
+    private final String filePath = System.getProperty("user.dir") + "/src/test/resources/testFile.jpg";
+
+    public void setSummaryTextField(String summaryValue) {
+        summary.sendKeys(summaryValue);
+
```
Comment (Rylan-cgi) [inline selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingEnvelopeChangeRequest.java:55]: Remove this space

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalCreateFundingEnvelopeChangeRequest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        driver = new ChromeDriver(options);
+        driver.manage().window().maximize();
+    }
+
+
+    @Test
+    public void CreateFundingEnvelopeChangeRequest() throws InterruptedException {
+        try {
+            driver.get(QA_PORTAL_URL);
+            //test = extent.createTest("Test - Create Funding Envelope Change Request");
```
Comment (Rylan-cgi) [inline selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalCreateFundingEnvelopeChangeRequest.java:30]: Uncomment this

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingAllocationTab.java (hunk 1 above)
Comment (Panika-cgi) [reply selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingAllocationTab.java]: done

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingEnvelopeChangeRequest.java (hunk 2 above)
Comment (Panika-cgi) [reply selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalFundingEnvelopeChangeRequest.java]: done

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalCreateFundingEnvelopeChangeRequest.java (hunk 3 above)
Comment (Panika-cgi) [reply selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalCreateFundingEnvelopeChangeRequest.java]: done

**Review by Rylan-cgi — APPROVED** [review]
Looks good to me

**Pull request merged**